docker/Dockerfile.dev

```diff
@@ -0,0 +1,14 @@
+FROM python:3.7
+
+ENV PYTHONUNBUFFERED=1
+ENV PYTHONDONTWRITEBYTECODE=1
+ENV PATH "/root/.poetry/bin:$PATH"
+RUN curl -sSL https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py | python
+
+WORKDIR /srv/search
+
+COPY ./pyproject.toml ./
+COPY ./poetry.lock ./
+
+RUN poetry config settings.virtualenvs.create false
+RUN poetry install
```

Comment:
Can reduce the number of layers in this file.

Reply:
How does that help?

Reply:
Grouping nearby `RUN` commands in a single one (concatenating the shell commands inside with `&&`) helps in general to reduce image size:
https://blog.replicated.com/refactoring-a-dockerfile-for-image-size/
In this case, there aren't files in the first `RUN` that could be removed in the second `RUN` and that hence should not be stored in the image layer resulting from that first `RUN`. However it is customary to always group the `RUN` commands whenever you can so that they don't look "off" from this pattern.

Reply:
Ok, I see now. I'll update the docker file.

Reply:
@giorgiosironi thanks for sending that article. I managed to reduce the image by 100mb having less layers and removing build dependencies.